### PR TITLE
Improve tests and fix some performance

### PR DESCRIPTION
**Pull request by maasencioh**
Unfinished work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,12 @@
     "test": "test"
   },
   "scripts": {
-    "test": "mocha --require should --reporter mocha-better-spec-reporter --recursive"
+    "eslint": "eslint src test",
+    "eslint-fix": "npm run eslint -- --fix",
+    "test": "npm run test-mocha && npm run eslint",
+    "test-cov": "istanbul cover node_modules/.bin/_mocha -- --require should --reporter dot --recursive",
+    "test-travis": "istanbul cover node_modules/.bin/_mocha --report lcovonly -- --require should --reporter dot --recursive",
```
Comment (targos) [inline package.json]: run eslint too

File: .travis.yml [hunk 2]
```diff
@@ -4,3 +4,5 @@ node_js:
   - lts/boron #v6
```
Comment (targos) [inline .travis.yml]: can you update the "node" element to "8" ?

File: src/index.js [hunk 3]
```diff
@@ -1,26 +1,24 @@
-'use strict';
+import {getSimilarity} from './getSimilarity';
+export {createTree} from './createTree';
 
-var extend = require('extend');
-var createTree = require('./createTree');
-var getSimilarity = require('./getSimilarity');
+export default function treeSimilarity(A, B, options) {
```
Comment (targos) [inline src/index.js]: missing default value for options

**Comment by maasencioh** [comment]
I'll reopen this when finish the refactor

**Pull request closed without merging**